dom/htmlelement-and-node.md

```diff
@@ -1,13 +1,33 @@
 # HTMLElement and Node
 
+The word **Node** can be referred back to Telecommunications Networks - _a device such as a computer or switch attached to a computer or telecommunications network, or a point in a network topology where lines intersect or terminate_. From there, it probably arrived in XML to represent any type of entity in a document e.g. a Comment node, a Text node, an Element node etc. Similarly, the representation of HTML DOM has various entities like tags, text content, comments, attributes etc. - all these are called **Nodes**.
```

Comment:
@KumarBhot  Beginning introduction is much better compared to earlier iteration. But can we do even better? Think from the perspective of DOM and how back-end engineers view DOM as compared to front-end engineers.